### DISCUSSION TITLE
Global Settings [ All Maps ]

### DISCUSSION THREAD

**Discussion by MSS970** [thurban/plugin_weathermap#80] [Q&A] [answered]
Hello,
I have started to use the "Weathermap" Beta 2 since yesterday and I've noticed the following notification in the Weathermap management page:
You are using the Boost plugin to update RRD files. Because this delays data being written to the files, it causes issues with Weathermap updates. You can resolve this by using Weathermap's 'poller_output' support, which grabs data directly from the poller. [You can enable that globally by clicking here].

I have tried to add a Global Setting, but I not sure about the value of the 2 required fiedls:

Name: (The name of the map-global SET variable)
Value: (What to set it to)

Could you please advise what set to in these 2 fields.

Thanks in advance.

**Comment by DatBoiTroy** [comment]
The one you need immediately should look like the following:

Name: rrd_use_poller_output
Value: 1

We have an additional global setting that I cannot remember exactly what it is doing, but might help later:

Name: nowarn_rrd_poller_output_aggregation
Value: 1

**Reply by TheWitness** [reply]
I did most of the interface, but did not make any dramatic changes with respect to settings.  Leave that for a future release.

**Reply by MSS970** [reply]
I have added the:
Name: rrd_use_poller_output
Value: 1

but the graphs have stopped showing any data!

Any suggestion please.

**Comment by TheWitness** [comment]
I need to remove that requirement.  In the mean time, it's a WeatherMap global and not a cacti.  The is a button at the top of the page for setting global variable.  Set it there.

**Comment by MSS970** [comment]
I have added the:
Name: rrd_use_poller_output
Value: 1

but the graphs have stopped showing any data!

Any suggestion please.

**Reply by TheWitness** [reply]
You mean in cacti general, or in Weathermap?

**Reply by MSS970** [reply]
In the Weathermap

**Comment by TheWitness** [comment]
I'm assuming you fixed the issue.

**Reply by MSS970** [reply]
Hey TheWitness,
Thanks a lot for your support man.

The weathermaps worked only after:

1. Deleting all the (Global, Group and Map parameter of Name: rrd_use_poller_output and Value: 1
2. Deleting the SET rrd_use_poller_output 1 from the weathermap conf file
3. Disabling the Boost (Configuration->Settings->Performance)

For some reason, the Boost does not work in my environment (Windows 2019 server, Cacti 1.2.23 and Weathermap 1.0 Beta 4 + fixes of issue#87 and issue#90), any suggestion to get the Boost working?!

Weathermap:
=========
--- develop ---

issue#87: WeatherMap Sort Order not respected on Top Tab
issue#90: Error: You have an error in your SQL syntax, Mysql 8.0 Weathermap 1.0
--- 1.0 Beta 4 ---

issue#64: Map Properties-Map Size-Unable to display entire map
issue#74: Weathermap Editor Issues
issue#75: There is no option to create a new map
issue#76: Graph Selector and Data Source Selector enhancements suggestions.
issue#77: Missing scroll bar on large map
issue#79: Bandwidth test a bit off with angled style and popup size issues
issue#81: Weathermap 1.0 - Editor button leads to the Editor page without css
issue#82: Unable to sort Maps

**Reply by TheWitness** [reply]
Enable boost and then open a command window after it's accumulated a few rows and then run it by hand.

```
php -q poller_boost.php --force --debug
```

Post the results from your log, errors and stats only.

**Comment by TheWitness** [comment, marked as answer]
Okay, Boost detection is now automatic.

**Comment by SubhojitRoy** [comment]
<img width="950" height="368" alt="Screenshot_4" src="https://github.com/user-attachments/assets/2ae9a5d6-f5f7-41f9-86de-c7541bc75806" />


hi there... 

i am trying to change mapstyle Link Labels from Percentage to Bits/sec but after i save it.... it becomes Percentage again.... but i can change it manually by using 
# DEFAULT definitions:
LINK DEFAULT
    BWLABEL bits

if anyone knows about it... it would be helpful
